### DISCUSSION TITLE
It can't find the class MinecraftServer

### DISCUSSION THREAD

**Discussion by evokerking1** [Minestom/Minestom#2451] [Help]
Code for the actual thing `package mc;

import net.minestom.server.*;
import org.jetbrains.annotations.NotNull;

public class mc_serve {

    public void mc_serve (@NotNull int port, @NotNull String ip) {
        // Initialize the server

        // Register Events (set spawn instance, teleport player at spawn)
        // Start the server

    }
}
`

My pom.xml `<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>dev.evokerking</groupId>
    <artifactId>Mc_Server</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <kotlin.code.style>official</kotlin.code.style>
        <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
    </properties>

    <repositories>
        <repository>
            <id>mavenCentral</id>
            <url>https://repo1.maven.org/maven2/</url>
        </repository>
    </repositories>

    <build>
        <sourceDirectory>src/main/kotlin</sourceDirectory>
        <testSourceDirectory>src/test/kotlin</testSourceDirectory>
        <plugins>
            <plugin>
                <groupId>org.jetbrains.kotlin</groupId>
                <artifactId>kotlin-maven-plugin</artifactId>
                <version>2.0.20</version>
                <executions>
                    <execution>
                        <id>compile</id>
                        <phase>compile</phase>
                        <goals>
                            <goal>compile</goal>
                        </goals>
                    </execution>
                    <execution>
                        <id>test-compile</id>
                        <phase>test-compile</phase>
                        <goals>
                            <goal>test-compile</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
            <plugin>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.22.2</version>
            </plugin>
            <plugin>
                <artifactId>maven-failsafe-plugin</artifactId>
                <version>2.22.2</version>
            </plugin>
            <plugin>
                <groupId>org.codehaus.mojo</groupId>
                <artifactId>exec-maven-plugin</artifactId>
                <version>1.6.0</version>
                <configuration>
                    <mainClass>MainKt</mainClass>
                </configuration>
            </plugin>
        </plugins>
    </build>

    <dependencies>
        <dependency>
            <groupId>net.minestom</groupId>
            <artifactId>minestom-lib-snapshots</artifactId>
            <version>lib-module-2b0274cee1</version>
        </dependency>

        <dependency>
            <groupId>org.jetbrains.kotlin</groupId>
            <artifactId>kotlin-test-junit5</artifactId>
            <version>2.0.20</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter</artifactId>
            <version>5.10.0</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.jetbrains.kotlin</groupId>
            <artifactId>kotlin-stdlib</artifactId>
            <version>2.0.20</version>
        </dependency>
        <dependency>
            <groupId>net.dv8tion</groupId>
            <artifactId>JDA</artifactId>
            <version>5.1.2</version>
        </dependency>
        
        <dependency>
            <groupId>io.github.revxrsal</groupId>
            <artifactId>lamp.common</artifactId>
            <version>4.0.0-beta.19</version>
        </dependency>

        
        <dependency>
            <groupId>io.github.revxrsal</groupId>
            <artifactId>lamp.jda</artifactId>
            <version>4.0.0-beta.19</version>
        </dependency>
        <dependency>
            <groupId>io.github.revxrsal</groupId>
            <artifactId>lamp.minestom</artifactId>
            <version>4.0.0-beta.19</version>
        </dependency>
    </dependencies>

</project>`

**Comment by evokerking1** [comment]
What's going on?

**Reply by xZeGoe** [reply]
You shouldn't use lib-module branch, use `minestom-snapshots` artifact id with latest commit hash (`dba90a461b`). For good, you can follow up the [wiki](https://minestom.net/docs/setup/your-first-server)
